kitsune/questions/jinja2/questions/mobile/new_question.html

```diff
@@ -1,92 +0,0 @@
-{% extends "mobile/base.html" %}
```

Comment:
Shouldn't this remove the ability to ask question through the mobile?

Reply:
Good point. This template should not have been removed. I've re-added it.